numpy/core/src/common/ufunc_override.c

```diff
@@ -40,17 +32,20 @@ get_non_default_array_ufunc(PyObject *obj)
     if ((PyObject *)Py_TYPE(obj) == ndarray) {
```

Comment:
As noted in the main comments, one can for some reason *not* replace this with `PyArray_CheckExact` - `PyArray_Type` does not seem to be defined here. I'll check if this is just missing an include...

Reply:
OK, think I found it - need `define _MULTIARRAYMODULE`.

Reply:
This is resolved?